axum/src/routing/method_routing.rs

```diff
@@ -178,8 +178,8 @@ macro_rules! chained_service_fn {
             ///     Ok::<_, Infallible>(Response::new(Body::empty()))
             /// });
             ///
-            /// // Requests to `GET /` will go to `service` and `POST /` will go to
-            /// // `other_service`.
+            /// // Requests to `GET /` will go to `other_service` and `POST /` will go to
+            /// // `service`.
```

Comment:
Maybe swap `GET` and `POST` instead of `service` and `other_service` (i.e. update the ordering within the sentence)? I think that would make this comment easier to follow.

Reply:
Agreed! Updated